tests/acceptance/ie11nocachehook.js

```diff
@@ -0,0 +1,31 @@
+import { RequestHook } from 'testcafe';
+
+/**
+ * IE11NoCacheHook solves the problem of IE11 caching all ajax requests
+ * by default by adding a 'no-store' header to requests in ie11.
+ *
+ * Without this hook, ajax requests cached by IE11 will be ignored
+ * testcafe's RequestLogger. This is only a problem in IE11.
```

Comment:
Just to make sure I understand, IE11 was caching the Ajax requests. This meant, on a page refresh, that a cached request was used. This wouldn't be caught by TestCafe's logger. Is that right?

Reply:
Yup! Also on back nav/forward nav. You can see this in browserstack in the ie11 network tab (how I originally found this) it'll say "from cache" or something like that.